index.html

```diff
@@ -50,13 +50,17 @@
   <!-- End Matomo Code -->
 </head>
 <body>
+	<div class="language-menu">
+		<button class="language-menu--btn">English ▼</button>
+		<a class="language-menu--item" href="/es/index.html">Spanish</a>
+	</div>				
```

Comment:
This component is technically a type of navigation, no? We should either move the div inside the `nav` element below or define another `nav` element. I thnk moving the `div` into the `nav` element below makes the most sense and is better for assistive technologies, such as screen readers. 

We have not talk about accessiblity, but we will be soon!